### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by nrmay**
Divers format and branding changes.
Added navigation bar.
Added phone/screen options.
Added pages: about, goals, minutes, links.
Added posts: SC minutes No. 1 & No. 2.

### CODE REVIEW

**Comment by nrmay** [comment]
RMIT Classification: Trusted

Hi Manodeep,

Can you approve the review changes?
As the pull request author I can't.

Cheers,

Nick.

________________________________

**Comment by manodeep** [comment]
@nrmay Will you please address couple of issues that I noted.

@nooriyahp Do you have any content to add in for the NZ side of the RSE survey?

File: _posts/2019-08-08-Australian-Results-2018-Survey.markdown [hunk 1]
```diff
@@ -5,14 +5,22 @@ date:   2019-08-09 12:00:00 +1000
...
+“The purpose of this survey is to collect information about people who 
+develop the software that is used in research. We call these people 
+Research Software Engineers (RSEs) <sup>[3]</sup>, but they exist under 
+many job titles (from postdoctoral researcher to applications consultant,
+ to bioinformatician).”
 
-There were 99 participants from Australia. The follow slides show some of the important data gathered from their responses: [Slides: Australian results of the 2018](/assets/RSE_Survey_2018_AU_v3.pdf)
+There were 99 participants from Australia. The follow slides show some 
+of the important data gathered from their responses: 
+[Slides: Australian results of the 2018](/assets/RSE_Survey_2018_AU_v3.pdf)
```
[the remark this answers is not in the document]
Comment (nrmay) [reply _posts/2019-08-08-Australian-Results-2018-Survey.markdown]: Suggest Nooriyah write a post for NZ results and add any slides she's presented on them.

File: _posts/2019-04-03-RSE-at-C3DIS.markdown [hunk 2]
```diff
@@ -2,18 +2,32 @@
...
+Because the main percentage of C3DIS delegates are researchers,
+we have decided to do a Hacky Hour to give back to the community. 
+The idea is to promote Hacky Hours generally, connect volunteer RSEs or 
+research support people who want to help, and to get some cross domain 
+problems written down.
 
-We then want to feed those cross domain problems into the Friday workshop to identify some solutions that we could convert into proposals to send to funders like the ARDC, NCI, Pawsey, and BPA. This is a great opportunity as these funders have just been provided funding and are already thinking of how they might allocate these funds.
+We then want to feed those cross domain problems into the Friday 
+workshop to identify some solutions that we could convert into proposals
+ to send to funders like the ARDC, NCI, Pawsey, and BPA.
```
[the remark this answers is not in the document]
Comment (nrmay) [reply _posts/2019-04-03-RSE-at-C3DIS.markdown]: done.

File: _posts/2019-08-20-RSE-mini-conference-at-eResearch-Australasia-2019.markdown [hunk 3]
```diff
@@ -2,27 +2,35 @@
...
+who define RSEs as: 
+“A growing number of people in academia combine expertise in programming
+ with an intricate understanding of research. Although this combination 
+ of skills is extremely valuable, these people lack a formal place in 
+ the academic system.” 
+ In Australasia, we interpret the term RSE inclusively to encompass:
 - academics and researchers who code;
 - professional software engineers working in the research space;
 - system administrators who maintain research systems and closely interact with their researchers; and
 - generalists who brings communities together and can engage with both researchers and technical staff.
```
[the remark this answers is not in the document]
Comment (nrmay) [reply _posts/2019-08-20-RSE-mini-conference-at-eResearch-Australasia-2019.markdown]: done

**Comment by manodeep** [comment]
Thanks @nrmay

**Pull request merged**